### ISSUE TITLE
Insufficient color contrast on Quiet Light theme

### ISSUE THREAD

**Issue by sapegin** [CompuIves/codesandbox-client#1703]
# 🐛 bug report

## Description of the problem

Light gray icons are barely visible:

![image](https://user-images.githubusercontent.com/70067/55475507-69732800-5614-11e9-94c0-35603e43d93a.png)

Light blue links have contrast ration of 1.45, minimum for WCAG AA is 4.5:

![image](https://user-images.githubusercontent.com/70067/55475566-932c4f00-5614-11e9-8a17-ba682008921b.png)

Contrast ratio on version numbers is 1.87:

![image](https://user-images.githubusercontent.com/70067/55475641-c4a51a80-5614-11e9-9d3f-599e0c2428b7.png)

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | PROD-1554195707-bfa032269 |
| Browser | Chrome |
| Operating System | macOS |
